beebop/app.py

```diff
@@ -147,51 +154,74 @@ def add_files(
 
 
 @app.errorhandler(500)
-def internal_server_error(e) -> json:
+def internal_server_error(e) -> tuple[Response, Literal[500]]:
     """
     :param e: [error]
-    :return json: [error response object]
+    :return Response: [error response object]
     """
+    logger.warning(f"Internal Server Error: {e}")
```

Comment:
This will log full exception information including the stack trace? So don't need to put it in the response any more (which now just gets the description)?